addons/sale_timesheet/migrations/12.0.1.0/post-migration.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Sergio Corato <https://github.com/sergiocorato>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def migrate_project_sale_order(env):
+    openupgrade.logged_query(
+        env.cr, """
+            UPDATE project_project p
+            SET sale_order_id = s.order_id
+            FROM sale_order_line s
+            WHERE s.id = p.sale_line_id
```

Comment:
```suggestion
            WHERE s.id = p.sale_line_id
            AND p.sale_order_id != s.order_id
```
and we use to name the alias for `sale_order_line` as `sol`, but that's a nitpicking